Contest.pas

```diff
@@ -861,6 +870,11 @@ procedure TContest.OnMeFinishedSending;
   z: integer;
   Dx : integer;
 begin
+  // reset Station ID counter after sending a CQ or 3 consecutive QSOs
+  if (msgCQ in Me.Msg) or
+     ((msgTU in Me.Msg) and (QsoCountSinceStationID >= 3)) then
```

Comment:
Is this 3 or more?

Reply:
Using `>=` is defensive coding in case the counter has somehow passed beyond 3. It should be reset to zero after it reaches 3, but in case, this will allow the counter to be reset. Thank you for asking.